Gemfile

```diff
@@ -63,10 +63,8 @@ gem "fine_uploader", path: "vendor/engines/fine_uploader"
 gem "fullcalendar", path: "vendor/engines/fullcalendar"
 gem "rubyzip"
 
-## controllers
-gem "prawn"
```

Comment:
`prawn-rails` includes these dependencies, so we don't need to explicitly include them ourselves anymore.